src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final int RESIZE_MULTIPLIER = 2;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_SIZE];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        Node<K, V> newNode = new Node<>(key, value);
+        int index = getIndex(key);
+        Node<K, V> node = table[index];
```

Comment:
seems like you either forgot to make this changes, either desided to pass on it :) but if you fixed it in "getValue()" method I think you should fix it there too

Reply:
I used it 2 times, so I think it is better to leave it as variable.